### PR TITLE
Added: Week 4 frontend task

### PR DESCRIPTION
**Pull request by basil08**

### CODE REVIEW

File: frontend/README.md [hunk 1]
```diff
@@ -90,3 +90,20 @@ ReactJS is made of components. The State of a component is an object that holds
 
 #### [Task](week3)
 Learn ReactJS, and make your website better using JSX components and State management. Bonus: improve the code quality too!
+
+### Week 4
+
+#### Concepts 
+
+- Integrating frontend with backend
```
Comment (as1605) [inline frontend/README.md:98]: Describe a bit more in 2-3 lines?

File: frontend/week4/README.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+The product model is given below along with their data types:
+
+```json
+ {
+    "title": "string",
+    "category": "string",
+    "price": 0,
+    "location": "string",
+    "description": "string",
```
Comment (as1605) [inline frontend/week4/README.md:42]: Added `img_url` and `metadata_user` fields
https://github.com/as1605/MarketPlace-Demo-Backend/commit/8d49742b91ac3402e9bafc0432cbcf43327ab9db

File: frontend/README.md [hunk 3]
```diff
@@ -90,3 +90,20 @@ ReactJS is made of components. The State of a component is an object that holds
...
+
+#### Resources  
+
+Resources and code snippets are shared in `week4/README.md` so check it out!      
+
+[0]: https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API
+[3]: https://pypi.org/project/python-socketio/
+[6]: https://socket.io/docs/v3/rooms/
+
+
```
Comment (as1605) [inline frontend/README.md:109]: Summarise the week4 task in 1 line as the banner title

File: frontend/week4/README.md (hunk 2 above)
Comment (basil08) [reply frontend/week4/README.md:42]: accha ok will update this also

**Pull request merged**